README.md

```diff
@@ -5,38 +5,99 @@
 
 This project is a modern implementation of the Ubuntu Desktop installer, using [subiquity](https://github.com/canonical/subiquity) as a backend and [Flutter](https://flutter.dev/) for the UI.
 
-## Checkout & Run
+![Preview Screenshot](.github/docs/images/preview_screenshot.png)
 
-Grab the source:
+## Availability
+
+The new Ubuntu Desktop installer is *planed* to ship with the next Ubuntu Desktop LTS 22.04 Impish Idri.
+
+## Test the new Ubuntu Desktop installer
```

Comment:
I'd call these titles `Test a live image` and `Test a local dry-run` or something like that